packages/sol-cov/src/source_maps.ts

```diff
@@ -0,0 +1,77 @@
+import * as _ from 'lodash';
+
+import { getPcToInstructionIndexMapping } from './instructions';
+import { LineColumn, LocationByOffset, SourceRange } from './types';
+
+const RADIX = 10;
+
+export interface SourceLocation {
+    offset: number;
+    length: number;
+    fileIndex: number;
+}
+
+export const getLocationByOffset = (str: string) => {
```

Comment:
Seems like you are now a fan of exporting individual functions. I guess I feel less particular about this then I used to... Do you feel strongly about it?

Reply:
Return type missing.